packages/gatsby-parcel-config/index.json

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
You'll want to check the `transformers` and remove more that won't be relevant for Gatsby code, e.g. pug or coffeescript